pizzabonus/pizzabonus.cpp

```diff
@@ -0,0 +1,74 @@
+#include <eosio/eosio.hpp>
+#include "./pizzabonus.h"
+
+#define DEFAULT_THRESHOLD 5;
+
+pizzabonus::pizzabonus(name self_contract, name first_receiver, eosio::datastream<const char*> ds) 
+: contract(self_contract, first_receiver, ds), ut(self_contract, self_contract.value) {
+  // TODO: Research passing custom parameters to the contract constructor
+  if (coupon_threshold == 0) {
+    coupon_threshold = DEFAULT_THRESHOLD;
+  }
```

Comment:
If you want to have custom parameters that can be changed, I think the best option is using singleton tables. Let me show you an example in this URL.
https://github.com/MitchPierias/advanced-eos-examples/tree/master/02_Singletons